doc/Language/faq.pod6

```diff
@@ -299,6 +299,69 @@ dependencies within a single compilation unit (e.g., file) are possible
 through stubbing. Therefore another possible solution is to move
 classes into the same compilation unit.
 
+=head2 System: How can I work with file and folder
+
+=item Delete: L<unlink|/routine/unlink> 'file.txt'; for a file, L<rmdir|/routine/rmdir> "dir"; for a directory
+
+=item Copy: 'file-from.twt'.L<copy|/type/IO::Path#routine_copy>('file-to.txt');
+
+=item Move: 'file-or-dir-from'.L<rename|/type/IO::Path#routine_rename>('file-or-dir-to');
+
+=item Create: 'file-or-dir-from'.L<rename|/type/IO::Path#routine_rename>('file-or-dir-to');
+
+=item Read: say 'file.txt'.IO.L<slurp|/type/IO::Path#routine_slurp>;
+
+=item List: say 'dir'.IO.L<dir|/type/IO::Path#routine_dir>;
+
+=item Inspect: say 'file.txt'.IO.L<absolute|/type/IO::Path#method_absolute>; say $filename.IO.L<extension|/type/IO::Path#method_extension>;
+
+=item Check: say 'exists' if 'file.txt'.IO.L<e|/type/IO::Path#method_e>;
+say 'is a file' if 'file.txt'.IO.L<f|/type/IO::Path#method_f>;
+say 'is a directory'  if 'dir'.IO.L<d|/type/IO::Path#method_d>;
+
+=item Write: 'file.txt'.IO.L<spurt|/type/IO::Path#method_spurt>: 'I ❤ 🦋'; Use the C<:append> adverb to append.
+
+=item Open: my $fh = 'file.txt'.IO.L<open|/type/IO::Path#method_open>;
+
+File handles are powerfull, read the L<overview|/language/io>, L<guide|/language/io-guide> or L<IO::Path|/type/IO::Path>. As usual, there are hundreds of ways to code with them, find yours!
+
+=head2 System: How can I recursively list file (find command)
```

Comment:
Not clear what you want to say, but grammar is incorrect. Also, not a FAQ if there's a simple module that can fix that.